packages/cra-template-typescript/template/gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.eslintcache
```

Comment:
I guess this would be fixed by #10177

Reply:
Updated in https://github.com/facebook/create-react-app/pull/10091/commits/4bf29906e5a8ba5ce9f0ae37c1d0207b8417fe28. Now some of the recovered tests will be temporarily broken but I think it's ok.